### ISSUE TITLE
contrib/cni/README.md: clarify which plugin versions are supported

### ISSUE THREAD

**Issue by dilyanpalauzov** [cri-o/cri-o#6075]
### What happened?

https://github.com/cri-o/cri-o/blob/main/contrib/cni/README.md#cni-plugin-installation-from-source contains:
> This tutorial will use the latest version of CNI plugins and build it from source.
> `git checkout v0.8.7`

Please elaborate which versions are supported by CRI-O and why 0.8.7 is suggested currently.

Later the file states:
> Output:
> `Building plugins`
> `    flannel`

`flannel` is not part of the the most recent https://github.com/containernetworking/plugins, see https://github.com/containernetworking/plugins/commit/77233dd79dc329e2214f0cad34 .  So it shall not be mentioned this way in that file.

### What did you expect to happen?

Not applicable.

### How can we reproduce it (as minimally and precisely as possible)?

Not applicable.

### Anything else we need to know?

Not applicable.

### CRI-O and Kubernetes version

Not applicable.

### OS version

Not applicable.

### Additional environment details (AWS, VirtualBox, physical, etc.)

Not applicable.

**Comment by haircommander** [comment]
yeah that is out of date. for 1.24.0, we recommend 1.1.1. same here, any desire to update?

**Comment by dilyanpalauzov** [comment]
I have no interest in opening a PR.
